emperor/support_files/js/controller.js

```diff
@@ -66,23 +94,39 @@ EmperorController.prototype.resize = function(width, height){
   // the area we have to present the plot is smaller than the total
   var plotWidth = this.$plotSpaceId.width();
 
+  // TODO: The below will need refactoring
+  // This is addressed in issue #414
```

Comment:
Can you remove this line? Also the phrasing of the comment above is a bit weird, but I'll leave that to you.